Apps/WebClient/src/Server/Startup.cs

```diff
@@ -118,6 +119,19 @@ public void ConfigureServices(IServiceCollection services)
             // Configure SPA
             services.AddControllersWithViews();
 
+            services.AddHsts(options =>
```

Comment:
Has this worked locally? I am guessing the https protocol will break vue service connection for hot reloading

Reply:
On the documentation link posted above the UserHsts method is surrounded by a check of the environment
```
if (!env.IsDevelopment())
        {
            // HSTS should only be enabled on production, not on localhost
            app.UseHsts();
        }

Reply:
It does not look like we have that on our code. When running locally, did Hot reloading worked on Vue?

Reply:
@FuriousLlama yes previously someone has already added it:
![image](https://user-images.githubusercontent.com/48332333/103950572-f56f8900-50f1-11eb-834f-939c9f05d701.png)

Reply:
for "When running locally, did Hot reloading worked on Vue?", yes it does for me.

Reply:
got it, thanks for checking!